kong/db/schema/init.lua

```diff
@@ -303,6 +323,30 @@ Schema.validators = {
     return nil, validation_errors.CONTAINS:format(wanted)
   end,
 
+  mutually_exclusive_subsets = function(value, subsets)
+    local subset_union = {} -- union of all subsets; key is element, value is the
+    for _, subset in ipairs(subsets) do -- the subset it's part of
```

Comment:
There is a typo or a missing word in the comments: `value is the` ... `the subset it's of`.

Reply:
Good :eyes:!